spec/factories/course_assessment_question_text_response_solutions.rb

```diff
@@ -9,10 +9,13 @@
     trait :keyword do
       solution_type :keyword
       solution 'Keyword'
+      grade 1
```

Comment:
this is the grade that will be awarded if the keyword is matched. Solutions are equivalent to "options" in MRQ.